gears/src/commands/node/run.rs

```diff
@@ -32,7 +32,7 @@ pub enum RunError {
     #[error("{0}")]
     HomeDirectory(String),
     #[error("{0}")]
-    Database(#[from] database::error::Error),
+    Database(String),
```

Comment:
Why did you replaced the type to `String`?

Reply:
Error type under `rockdb` feature